frontend/src/components/common/input/input.tsx

```diff
@@ -16,9 +16,10 @@ type Props = {
   label: string;
   name: FormControlPath;
   placeholder?: string;
-  type?: 'text' | 'email' | 'date' | 'password';
+  type?: 'text' | 'email' | 'date' | 'password' | 'textarea';
   hasVisuallyHiddenLabel?: boolean;
   inputClassName?: string;
+  textAreaRows?: number;
```

Comment:
```suggestion
 rows?: number;
```
if we have a `row` prop. it should be a text-area